client/src/components/editMenuItems.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import "../app.css";
+
+import Placeholder from "../resources/images/150.png";
+import Toppings from "./toppings";
+import Topping from "./topping";
+
+class MenuItemEditor extends Component {
+    render() {
+        return (
+            <form>
+                <div
+                    className="form-group bg-light"
+                    style={{ height: "40rem", width: "60rem" }}
+                >
+                    <div style={{ height: "35%" }}>
+                        <div
+                            id="upperLeft"
+                            className="p-1 w-50 h-100  text-white float-left"
+                        >
+                            <input
+                                className="form-control form-control-lg w-50 m-3"
+                                type="text"
+                                placeholder="Namn..."
+                            ></input>
+                            <textarea
+                                className="form-control form-control-lg w-75 h-50 m-3 "
+                                type="text"
+                                placeholder="Beskrivning..."
+                            ></textarea>
+                        </div>
+                        <div
+                            id="upperRight"
+                            className=" w-25 h-100 float-right mr-5"
+                        >
+                            <img
+                                src={Placeholder}
+                                className="rounded mt-4"
+                                alt="Choose portrait"
+                            />
+                            <input
+                                type="file"
+                                className="mt-1"
+                                id="productImage"
+                            ></input>
+                        </div>
+                    </div>
+                    <Toppings></Toppings>
+                    <div>
+                        <input
+                            className="ml-4 mt-3 mr-1 w-25 form-control-sm"
+                            type="text"
+                            placeholder="Pris utan tillägg..."
+                        ></input>
+                        <span>{"(kr)"}</span>

```

Comment:
Doesn't need to be seen as js
```suggestion
                        <span>(kr)</span>

```